index.html

```diff
@@ -245,8 +245,8 @@
             return m("pre.f6", {
               style: "white-space: pre-wrap; tab-size: 4;",
             }, diff.map(p => {
-                const c = p.added ? 'green' : p.removed ? 'red' : '';
-                return m('span', {class: c}, p.value);
+                const c = p.added ? 'bg-washed-green dark-green' : p.removed ? 'bg-washed-red dark-red' : '';
+                return p.value.split("\n").flatMap(l => [m('span', {title: "newline", class: c || "moon-gray", style: "user-select: none"}, "↵"), "\n", m('span', {class: c}, l)]).slice(2);
```

Comment:
- added red / green background (useful when the text is just a space for example)
- added newline symbols to the diff